src/pages/search.js

```diff
@@ -16,6 +27,12 @@ export default function Search() {
                 baseId="search"
                 searchTerm={router.query?.searchTerm}
                 filter={filter}
+                selectedTab={tab}
+                onTabSelectionChange={(event, selection) => {
+                    router.push(
+                        `/${NavigationConstants.SEARCH}?searchTerm=${searchTerm}&filter=${filter}&selectedTab=${selection}`
+                    );
+                }}
             />
         </>
     );
```

Comment:
`selectedTab ` query param enables users to come back to the same tab they were in after navigating away from search results and then when hit browser back button.